lib/node_modules/@stdlib/stats/base/dists/rayleigh/cdf/benchmark/benchmark.js

```diff
@@ -32,15 +33,22 @@ var cdf = require( './../lib' );
 
 bench( pkg, function benchmark( b ) {
 	var sigma;
+	var len;
 	var x;
 	var y;
 	var i;
 
+	len = 100;
+	x = new Float64Array( len );
+	sigma = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = ( randu() * 100.0 ) - 100;
```

Comment:
While not present in the original, let's add decimal point here to signal that we are working with floating-point numbers here:

```suggestion
		x[ i ] = ( randu() * 100.0 ) - 100.0;
```